### ISSUE TITLE
[Warning][Django1.10]RemovedInDjango110Warning 

### ISSUE THREAD

**Issue by SalahAdDin** [digi604/django-smart-selects#107]
Hi man, now i test your packages and i have the next errors:

```
/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/smart_selects/urls.py:9: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got filterchain_all). Pass the callable instead.
  'filterchain_all', name='chained_filter_all'),

/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/smart_selects/urls.py:11: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got filterchain). Pass the callable instead.
  'filterchain', name='chained_filter'),

/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/smart_selects/urls.py:13: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got filterchain). Pass the callable instead.
  'filterchain', name='chained_filter'),

/run/media/salahaddin/Data/Proyectos/Trabajo/ccct/lib/python3.5/site-packages/smart_selects/urls.py:13: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  'filterchain', name='chained_filter'),
```

Thank you very much man for your work.

**Comment by DiogoMarques29** [comment]
@SalahAdDin there is a issue about this already, see #104 
This will be merge in a few days like @jimmykobe1171 said.

best regards

**Comment by SalahAdDin** [comment]
:+1: Thanks

**Comment by jimmykobe1171** [comment]
fixed and release to 1.2.1 :)
